scripts/compare_perf.py

```diff
@@ -119,7 +134,7 @@ def main() -> None:
         f"Min: {min_rel_dur:.3f}x, "
         f"Max: {max_rel_dur:.3f}x"
     )
-    print(f"Total Baseline: {total_baseline} s, Latest: {total_latest} s")
+    print(f"Total Baseline: {total_baseline:.3f} s, Latest: {total_latest:.3f} s")
```

Comment:
below there is "See run output for more details."
But in the bot-actions messages I get, where do I find this "run output"?
A link would be great.

Reply:
It doesn't seem to be possible to create a link without using the GitHub API, which is overkill. I'll change the message a bit.